contact.csv (3).ipynb

```diff
@@ -0,0 +1,173 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25264925",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b312719",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file=open('contact.csv','w',newline='')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "66511340",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w=csv.writer(file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0686a469",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "header=['Name','Contact']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b3c31531",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w.writerow(header)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2a16388a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "List=[]\n",
+    "for i in range(5):\n",
+    "    name=input(\"Enter person name:\")\n",
+    "    contact=int(input(\"Enter person contact number:\"))\n",
+    "    l=[name,contact]\n",
+    "    List.append(l)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "cdea2f10",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w.writerows(List)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "5b862205",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# file.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "21c546ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file=open('contact.csv','r',newline='')\n"
```

Comment:
How to Open This File To another way ?

Reply:
to open another way we can use with open('filename.csv','w',newline='') as f: